### ISSUE TITLE
Problem of NAN

### ISSUE THREAD

**Issue by kjy09** [csguoh/MambaIR#34]
I encountered a problem of NAN after training for several epochs. I tried to reduce the learning rate to address this issue. However, it worked only for several epochs and the loss value became NAN again. Do you have any experience in addressing this problem?

**Comment by csguoh** [comment]
Hi,
Which task are you training for?  The RMSNorm may help.

**Comment by kjy09** [comment]
Thank you for the reply.
